.github/workflows/main.yml

```
@@ -22,16 +32,12 @@ jobs:
-
-      - name: Deploy (placeholder)
+      - name: Build and uploard docker image
         run: |
-          echo "Deploy step placeholder. Add your deployment commands or actions here."
+          docker build . --tag vivematcher
+          docker login ghcr.io -u $GITHUB_ACTOR -p $PAT_TOKEN
+          docker tag vivematcher ghcr.io/$REPO_OWNER/vivematcher:latest
+          docker push ghcr.io/$REPO_OWNER/vivematcher:latest
+
+      - name: Rollout
+        run: sshpass -p "$SSH_PASSWORD" ssh -o StrictHostKeyChecking=no $SSH_USER@$SSH_IP -p $SSH_PORT 'sudo kubectl rollout restart deployment vivematcher-app'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)